2_process/src/process.r

```diff
@@ -0,0 +1,40 @@
+library(dplyr)
+library(stringr)
+library(whisker)
+
+process_data<-function(data_file='model_RMSEs.csv', processed_data_file='model_summary_results.csv', diagnostic_output_file = 'model_diagnostic_text.txt'){
+	data_file_path = file.path("1_fetch","out",data_file)
```

Comment:
try a ctrl-a, ctrl-shift-i to get standard indentation in this file

Reply:
Thanks, and done.